user-interface/src/lib/components/Pill.scss

```diff
@@ -12,10 +12,19 @@
     height: 1.5rem;
     border: 2px solid #949494;
     border-radius: 0.75rem;
+    max-width: 100%;
     min-width: 4rem;
     padding: 0 0.25rem 0 0.6rem;
     margin: 0.25rem;
     cursor: pointer;
+
+    .pill-text {
+      display: inline-block;
+      white-space: nowrap;
+      text-overflow: ellipsis;
+      overflow: hidden;
+      max-width: calc(100% - 1em);
```

Comment:
`100% - 1rem`?

Reply:
I used 1em because that is what is used by the icon from uswds.  Was done on purpose.